tabbycat/api/serializers.py

```diff
@@ -2,20 +2,74 @@
 
 from breakqual.models import BreakCategory
 from participants.models import Speaker, SpeakerCategory
+from tournaments.models import Tournament
+
+from .fields import TournamentHyperlinkedIdentityField
+
+
+class TournamentAtRootSerializer(serializers.HyperlinkedModelSerializer):
+
+    url = serializers.HyperlinkedIdentityField(
+        view_name='api-tournament-detail',
+        lookup_field='slug', lookup_url_kwarg='tournament_slug')
+
+    class Meta:
+        model = Tournament
+        fields = ('name', 'short_name', 'slug', 'url', 'active')
+
+
+class TournamentEndpointsSerializer(serializers.Serializer):
+
+    break_categories = serializers.HyperlinkedIdentityField(
+        view_name='api-breakcategory-list',
+        lookup_field='slug', lookup_url_kwarg='tournament_slug')
+    speaker_categories = serializers.HyperlinkedIdentityField(
+        view_name='api-speakercategory-list',
+        lookup_field='slug', lookup_url_kwarg='tournament_slug')
+
+
+class TournamentSerializer(serializers.ModelSerializer):
+
+    urls = TournamentEndpointsSerializer(source='*', read_only=True)
```

Comment:
From what I see, we should prefix this kind of field with an underscore to show this field is not in the data structure but "meta" or for the API use.

Reply:
I also suggest adding a `url` field to `api-tournament-detail` with the slug